src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,49 +1,134 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final int DEFAULT_CAPACITY = 10;
+    private static final Object[] EMPTY_DATA = {};
+    private static final int ZERO = 0;
+    private static final int ONE = 1;
+    private T[] elements;
+    private int size;
+
+    public ArrayList() {
+        this.elements = (T[]) new Object[DEFAULT_CAPACITY];
+        this.size = ZERO;
```

Comment:
What is the default value of `int` variables?

Reply:
Zero, but I thought that my compiler would say to me: "initialize variable ..". Fixed my error